google.com/src/ada_google_com/env.py

```diff
@@ -0,0 +1,17 @@
+"""
+Keep here all env imports
+"""
+import os
+from pathlib import Path
+
+_value = os.getenv('ADA_LEARNING_GOOGLE_COM_CLIENT_SECRETS_FILE')
+CLIENT_SECRETS_FILE = Path(_value).resolve() if _value else None
+
+_value = os.environ['ADA_LEARNING_GOOGLE_COM_CONFIG']
+assert _value, 'env ADA_LEARNING_GOOGLE_COM_CONFIG has to be a path'
```

Comment:
it's recommended to use exceptions instead of assert in code outside testing.

Reply:
fixed